### PR TITLE
Fixes blood brother on the antag panel.

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Makes blood brother on the antag panel work again.

image

## Why It's Good For The Game

It doesnt work and stopped me from using antag tokens. 😔 

## Changelog
:cl:
fix: Fixes blood brothers admin add
/:cl:

### CODE REVIEW

File: code/modules/antagonists/brother/brother.dm [hunk 1]
```diff
@@ -84,10 +84,12 @@
 	var/choice = input(admin,"Choose the blood brother.", "Brother") as null|anything in sortNames(candidates)
 	if(!choice)
 		return
-	var/datum/mind/bro = candidates[choice]
+	var/datum/mind/bro = choice
 	var/datum/team/brother_team/T = new
 	T.add_member(new_owner)
 	T.add_member(bro)
+	bro.special_role = "brother"
+	new_owner.special_role = "brother"
```
Comment (Archanial) [inline code/modules/antagonists/brother/brother.dm]: This is handled in add antag datum I'm pretty sure

**Review by Archanial — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/brother/brother.dm (hunk 1 above)
Comment (Archanial) [reply code/modules/antagonists/brother/brother.dm]: actually it seems like `add_member` is also handled by antag datum

Comment (PowerfulBacon) [inline code/modules/antagonists/brother/brother.dm]:
```suggestion
```

Comment (Archanial) [inline code/modules/antagonists/brother/brother.dm]:
```suggestion
```

**Pull request merged**